lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/FeatureService.java

```diff
@@ -76,10 +80,10 @@ private FeatureService() {
    * @param libertyVersion - version of Liberty to fetch features for
    * @return list of features supported by the provided version of Liberty
    */
-  private List<Feature> fetchFeaturesForVersion(String libertyVersion) throws IOException, JsonParseException {
-    String featureEndpoint = String.format(
-        "https://repo1.maven.org/maven2/io/openliberty/features/features/%s/features-%s.json", libertyVersion,
-        libertyVersion);
+  private List<Feature> fetchFeaturesForVersion(String libertyVersion, String libertyRuntime) throws IOException, JsonParseException {
+    String featureEndpoint = libertyRuntime.equals("wlp") ? String.format(wlpFeatureEndpoint, libertyVersion, libertyVersion) : 
```

Comment:
I think the *FeatureEndpoint strings can be modified so we only need to pass `libertyVersion` once instead of twice in the `String.format()` args

Reply:
How? That part of the code did not change. So I was not looking at how to optimize it.

Reply:
`%1$s` should do it